recipe.json

```diff
@@ -55,7 +55,7 @@
           "URI": "docker:$DOCKER_IMAGE"
         },
         {
-          "URI": "s3://$BUCKET_NAME/$COMPONENT_VERSION/home-assistant.zip",
+          "URI": "s3://BUCKET_NAME/COMPONENT_VERSION/home-assistant.zip",
```

Comment:
As long as the uri starts with s3:// and ends with artifact name, gdk can replace the s3 URI (Eg. `s3://home-assistant.zip` i.e what you had earlier is still valid.

Reply:
OK. I have $DOCKER_IMAGE and $SECRET_ARN as two variables my custom build will populate. I dropped the dollar signs here to distinguish my variables from what GDK would do.